src/SbManager/Scripts/manager/statusController.js

```diff
@@ -0,0 +1,20 @@
+﻿$app.controller('statusController', ['$scope', function ($scope) {
+    $scope.refresh = function () {
+        $scope.model = null;
+        $.ajax({
+            url: window.applicationBasePath + "/api/v1/busmanager/",
+            dataType: 'json',
+            success: function (d) {
+                $scope.model = d;
+                var currentTime = new Date();
+                $scope.time = currentTime.getHours() + ":" + currentTime.getMinutes();
+                $scope.$digest();
```

Comment:
As the callback leaves the angular scope we need to trigger a digest cycle here otherwise the ng-if on the respective page that watches model will not trigger.